tests/chocolatey-tests/commands/choco-install.Tests.ps1

```diff
@@ -1502,11 +1503,14 @@ To install a local, or remote file, you may use:
         }
     }
 
-    Context "Installing package while specifying a cache location (Arg: <_>)" -ForEach '-c', '--cache', '--cachelocation', '--cache-location' -Tag Internal, Testing {
+    # We are marking this test as internal, as the package we need to make use
+    # of downloads a zip archive from a internal server, and the package is also
+    # only located on an internal feed.
+    Context "Installing package while specifying a cache location (Arg: <_>)" -ForEach '-c', '--cache', '--cachelocation', '--cache-location' -Tag Internal, LongPaths, CacheLocation {
         BeforeAll {
             $paths = Restore-ChocolateyInstallSnapshot
 
-            $Output = Invoke-Choco install install-chocolateyzip --version 3.21.2 --confirm "$_" "$($paths.CachePathLong)" --verbose
+            $Output = Invoke-Choco install install-chocolateyzip --version 3.21.2 --confirm "--cache" "$($paths.CachePathLong)" --no-progress

```

Comment:
Why is this replacing the passed in argument with `--cache`?

Reply:
Used it during testing and forgot to remove it again afterwards.

Good catch, I have fixed that call.